Cargo.toml

```diff
@@ -33,7 +33,7 @@ crossbeam-channel = "0.5.14"
 csv = "1.3.1"
 ctrlc = "3.4.5"
 dashmap = "6.1.0"
-deno_task_shell = "0.24.0"
+deno_task_shell = "0.26.0"
```

Comment:
I remember this introducing an issue @Hofer-Julian do you have top of mind whether we could do this?

Reply:
I'm mistaken, I've tested what I can think of, and it doesn't seem affected. Sorry for the disturbance.